src/ui/pages/work/Workorder.jsx

```diff
@@ -341,14 +342,17 @@ const Workorder = () => {
         label: "NCRs",
         isVisibleWhenNewEntity: false,
         maximizable: true,
-        render: () => (
-            <NCRIframeContainer
+        render: () => ( 
+          isValidHttpUrl(`${applicationData.EL_TBURL}/ncr`) 
```

Comment:
Imo we should not validate that the parameter is a valid url, only that the value is preset. If the user configures an invalid URL then we fail silently, and the user is none the wiser as the AssetNCRs will keep appearing.